### DISCUSSION TITLE
chatgpt api是收费了吗？

### DISCUSSION THREAD

**Discussion by QingYu-sunshine** [MuiseDestiny/zotero-gpt#213] [Q&A] [answered]
之前一直用的好好的，今天想起来用，突然变成这个样子了，显示额度已经用光了。请问chatgpt api是收费了吗？有懂得大佬能给解答一下吗？
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/123077881/6f904220-c37e-4505-acb6-408d311e03f9)

**Comment by MuiseDestiny** [comment, marked as answer]
它一直是收费的。从诞生之初。

**Reply by chunweizhu** [reply]
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/47244527/de8cbe65-e066-42b4-ad0b-6a2cfc55b992)
同样的问题: `insufficient_quota`
似乎额度还没有用完，请问这是什么原因?

**Comment by cocoper** [comment]
我也是同样的问题，额度还有
